README.md

```diff
@@ -13,17 +13,23 @@ You can run mlcp across many threads on a single machine or across many nodes in
 
 The MarkLogic Connector for Hadoop is an extension to Hadoop’s MapReduce framework that allows you to easily and efficiently communicate with a MarkLogic database from within a Hadoop job. From 10.0-5, Hadoop Connector is removed from a separate release, but mlcp still uses Hadoop Connector as an internal dependency.
 
-## Release Note
+## Release Notes
+
+### What's New in mlcp and Hadoop Connector 11.1.0
+- Now requires JRE 11 or later.
+- Upgraded Jena libraries from 2.13.0 to 4.9.0 to mitigate security vulnerability.
+- Upgraded Jackson, Hadoop, Xstream, and Guava libraries to mitigate security vulnerabilities.
+- Added support for reverse proxy and connection to MarkLogic Cloud. 
```

Comment:
Hi Abika,

Can you please move this line to the second?

Reply:
Changed the order. Thanks..